archive/archivist/trigger_archive_bag.py

```diff
@@ -3,23 +3,28 @@
 """
 Create a request to archive a bag
 
-Usage: trigger_archive_bag.py <BAG>... [--bucket=<BUCKET_NAME>]  [--topic=<TOPIC_NAME>]
+Usage: trigger_archive_bag.py [sns] <BAG>... [--bucket=<BUCKET_NAME>]  [--topic=<TOPIC_NAME>|--api=<API>]
```

Comment:
minor: I think a flag `--sns=(true|false)` would be more idiomatic, but that's not a strong feeling.